avanza/market/client.go

```diff
@@ -40,3 +41,9 @@ func (a *MarketClient) GetMarketData(ctx context.Context, params *models.GetMark
 	err := a.Call(ctx, http.MethodGet, GetMarketDataPath, params, res, options...)
 	return res, err
 }
+
+func (a *MarketClient) GetChartData(ctx context.Context, params *models.GetMarketDataParams, options ...models.RequestOption) (*models.GetChartDataResponse, error) {
```

Comment:
Add a seperate `models.GetChartDataParams` instead of reusing GetMarketDataParams.
Also we should add `timePeriod` and `resolution` enum/consts and add them as optional parameters. Day and year could be default options.

Available chart options can be seen in my typescript package
https://github.com/AnteWall/avanza-ts/blob/main/src/client/price_chart_types.ts#L9